dace/frontend/fortran/fortran_parser.py

```diff
@@ -1028,845 +1029,25 @@ def subroutine2sdfg(self, node: ast_internal_classes.Subroutine_Subprogram_Node,
 
             sdfg_name = self.name_mapping.get(sdfg).get(ast_utils.get_name(variable_in_call))
```

Comment:
As far as I can tell: `substate_sources`, `substate_destinations`, `all_arrays`, `sdfg_name`, `globalsdfg_name`, `matched`, `view_ranges` are all unused code (and computing `sdfg_name` will fail on code like `call fun(x = 1.1)`, which will also happen in `process_variable_call()` even if you remove these unused variables).

Reply:
I've added a couple of commits to address this.